src/js/util.js

```diff
@@ -275,43 +275,25 @@ var Util;
         },
 
         getSelectionRange: function (ownerDocument) {
-            var selection = ownerDocument.getSelection();
-            if (selection.rangeCount === 0) {
-                return null;
-            }
-            return selection.getRangeAt(0);
+            this.deprecated('Util.getSelectionRange', 'Selection.getSelectionRange', 'v5.0.0');
+
+            return Selection.getSelectionRange(ownerDocument);
```

Comment:
This might be a bit dangerous, I believe Util will be defined before Selection...